apps/codeforafrica/src/pages/_app.page.js

```diff
@@ -17,6 +18,21 @@ function MyApp(props) {
 
   return (
     <>
+      <Script
+        strategy="lazyOnload"
```

Comment:
```suggestion
```

Should use `afterInteractive` (the default strategy) for analytics. [Docs](https://nextjs.org/docs/app/api-reference/components/script#afterinteractive).